sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -822,6 +822,54 @@ public void StartRecognizeReceiptsFromUriThrowsForNonExistingContent()
             Assert.AreEqual("FailedToDownloadImage", ex.ErrorCode);
         }
 
+        [Test]
+        [TestCase("en-US")]
+        [TestCase("")]
+        public async Task StartRecognizeReceiptsWithSupportedLocale(string locale)
+        {
+            var client = CreateFormRecognizerClient();
+            var options = new RecognizeReceiptsOptions()
+            {
+                IncludeFieldElements = true,
+                Locale = locale
+            };
+            RecognizeReceiptsOperation operation;
+
+            using var stream = FormRecognizerTestEnvironment.CreateStream(TestFile.ReceiptJpg);
+            using (Recording.DisableRequestBodyRecording())
+            {
+                operation = await client.StartRecognizeReceiptsAsync(stream, options);
+            }
+
+            RecognizedFormCollection recognizedForms = await operation.WaitForCompletionAsync(PollingInterval);
+
+            var receipt = recognizedForms.Single();
+
+            ValidatePrebuiltForm(
+                receipt,
+                includeFieldElements: true,
+                expectedFirstPageNumber: 1,
+                expectedLastPageNumber: 1);
+
+            Assert.Greater(receipt.Fields.Count, 0);
+
+            var receiptPage = receipt.Pages.Single();
+
+            Assert.Greater(receiptPage.Lines.Count, 0);
+            Assert.AreEqual(0, receiptPage.SelectionMarks.Count);
+            Assert.AreEqual(0, receiptPage.Tables.Count);
```

Comment:
Did we miss validating the sent locale in this test case?

Reply:
Here I am testing that the response we get from the service is valid and that all the fields contain information as expected. In the FormRecognizerClientMockTests is where I am checking that what the user specified is what we are sending.

I am making this division because the logic to intercept the pipeline before sending the request is in the Mocking classes.